app/models/rental.rb

```diff
@@ -29,6 +29,7 @@ class Rental < ActiveRecord::Base
   scope :upcoming_rentals, -> { reserved.where('start_time <= ? AND end_time >= ?', DateTime.current + 1.day, DateTime.current) }
   scope :all_future_rentals, -> { reserved.where('end_time >= ?', DateTime.current) }
   scope :no_show_rentals, -> { reserved.where('end_time < ?', DateTime.current) }
+  scope :inactive_rentals, -> { where('rental_status = ? OR rental_status = ?', 'canceled', 'checked_in') }
```

Comment:
@phantompatrol , not sure if there's a better way than using the strings along with the above syntax. I have access to the state methods individually but can't find an 'or' method where the scope will choose between two states. i.e. I can call a scope with '{ canceled }' and it will return the canceled rentals but not '{ canceled || checked_in}'